LocalSettings.php

```diff
@@ -393,6 +393,7 @@
 			'speleo.wiki' => 'speleowiki',
 			'spiral.wiki' => 'spiralwiki',
 			'takethatwiki.com' => 'takethatwikiwiki',
+			'template.wiki' => 'templatewiki',
```

Comment:
A cert would need to be installed and the domain registered and pointed to miraheze for this to work